index.js

```diff
@@ -128,6 +128,10 @@ var appServer = function(config) {
       });
 
       var endpoint = path.posix.join(normalizedRoot, app.name);
+	  self.express.route(endpoint+'/appid').get(function(req, res){
```

Comment:
Something wrong with tabs/spaces here at least.

Now do we really want this with every app server?